docs/docs/91-migrations.md

```diff
@@ -12,15 +12,36 @@ Some versions need some changes to the server configuration or the pipeline conf
 
   Read more about it at the [Project Settings](/docs/usage/project-settings#pipeline-path)
 
+- From version `0.15.0` ongoing there will be three types of docker images: `latest`, `next` and `x.x.x` with an alpine variant for each type like `latest-alpine`.
+  If you used `latest` before to try pre-release features you should switch to `next` after this release.
+
 - Dropped support for `DRONE_*` environment variables. The according `WOODPECKER_*` variables must be used instead.
   Additionally some alternative namings have been removed to simplify maintenance:
   - `WOODPECKER_AGENT_SECRET` replaces `WOODPECKER_SECRET`, `DRONE_SECRET`, `WOODPECKER_PASSWORD`, `DRONE_PASSWORD` and `DRONE_AGENT_SECRET`.
   - `WOODPECKER_HOST` replaces `DRONE_HOST` and `DRONE_SERVER_HOST`.
   - `WOODPECKER_DATABASE_DRIVER` replaces `DRONE_DATABASE_DRIVER` and `DATABASE_DRIVER`.
   - `WOODPECKER_DATABASE_DATASOURCE` replaces `DRONE_DATABASE_DATASOURCE` and `DATABASE_CONFIG`.
 
-- From version `0.15.0` ongoing there will be three types of docker images: `latest`, `next` and `x.x.x` with an alpine variant for each type like `latest-alpine`.
-  If you used `latest` before to try pre-release features you should switch to `next` after this release.
+- Dropped support for `DRONE_*` environment variables in pipeline steps. Pipeline meta-data can be accessed with `CI_*` variables.
+  - `CI_*` prefix replaces `DRONE_*`
+  - `CI` value is now `woodpecker`
+  - `DRONE=true` has been removed
+  - Some variables got deprecated and will be removed in future versions. Please migrate to the new names. Same applies for `DRONE_` of them.
+    - CI_ARCH => use CI_SYSTEM_ARCH
+    - CI_COMMIT => CI_COMMIT_SHA
+    - CI_REMOTE_URL => use CI_REPO_REMOTE
```

Comment:
as i understand the pull we still have no DRONE_ fallback

that would make all existing plugins using them useless!

Reply:
we dont have to promote them and I would add them in a seperate func (even seperated in a own go file) but It should be there